internal/cmd/task/task.go

```diff
@@ -0,0 +1,99 @@
+package task
+
+import (
+	"context"
+
+	"github.com/influxdata/influx-cli/v2/internal/api"
+	"github.com/influxdata/influx-cli/v2/internal/cmd"
+)
+
+type Client struct {
+	cmd.CLI
+	api.TasksApi
+	// AllowEmptyOrg will be useful for Kapacitor which doesn't use org / orgID
+	AllowEmptyOrg bool
```

Comment:
Hmm, on the `Client` because it'll be useful across commands?

Reply:
Each command seems to deal with org indepedently - I wouldn't want to put this on the client unless I re-evaluated all the other org fetching code, which seems like a bigger change.

Reply:
Ah, let me rephrase: it's on `task.Client` instead of `task.CreateParams` because it'll be useful for other task functions?